andela/metis-ah-frontend
# #161050935 Connect landing page to backend to receive live data

#### What does this PR do?
Connects the Landing page to receive live data from the Metis-AH api
#### Description of Task to be completed?
Create thunks for handling asynchronous calls to the api 
Create actions for updating featured articles and authors in the global state
Update convertLikes function to make conversation decimal accurate
Update reducers to update global state appropriately
Write tests to test landing page rendering

#### How should this be manually tested?
Clone the repo, run `npm install`, run `npm run dev`, visit your browser to see the Landing page

#### What are the relevant pivotal tracker stories?
#161050935
#### Screenshots (if appropriate)
<img width="1354" alt="screen shot 2018-10-14 at 8 08 40 pm" src="https://user-images.githubusercontent.com/42326622/46921493-f3dab780-cff3-11e8-9921-a85dc4a89e36.png">
<img width="1420" alt="screen shot 2018-10-13 at 11 39 16 pm" src="https://user-images.githubusercontent.com/42326622/46921496-fb01c580-cff3-11e8-8a99-324106654179.png">
